### DISCUSSION TITLE
Salvaging the Exodii

### DISCUSSION THREAD

**Discussion by DoubleBullfrog** [cataclysmbn/Cataclysm-BN#5199] [General]
This will literally never happen because it's too much work for too little benefit and the concept of Exodii is tainted, but I need to get this out of my head or it will bother me forever.
	
1. Keep the current CBM system; autodocs, anaesthetic, salvaging from appropriate zombies, etc. The Exodii will not immediately install early bionics for you for multiple reasons.

- 	A. They don't use, know how to make, or want to use or know how to make anaesthetic for cultural reasons; if you want them to install bionics for you, you'll need to buy the Sensory Dulling CBM and install it yourself. They will sell you said CBM but they need to trust you first, which means you need to do quests for them.
- 	B. Their bionic power storage and wiring standards are initially incompatible with local standard. You'll need to bring them functional examples of both a local (generic brand) bionic power storage and generation system so they can figure out how to make their bionics compatible with Earth bionics. They are very willing to do this because the chance to access a massive amount of salvagable CBMs is too good to pass up, most Earths don't have anywhere near their level of technology.
- 	C. The Exodii just don't have equivalents of most of the CBMs early characters would find or want. They don't use "convenience" bionics like Cranial Flashlights and Finger Lighters or early power CBMs like Battery Systems and Joint Torsion Ratchets. Mostly they use Cable Charger Systems for their worker drones and Micro Reactors for their combat models, and purpose-built robot bodies for industrial or combat scenarios.

2. A few weird CBMs can be moved to Exodii-only, all of the rest remain findable in the world. To facilitate this, add bionic manufacturers to CBM descriptions for lore and player readability reasons, just a JSON tag and a short line of text in the description. Manufacturer is NOT itemgroup; for example, military sources can also drop generic or industry CBMs if appropriate e.g. soldier/bio-op Zs dropping power generators, storage, Skeletal Bracing, Air Filtration, that sort of thing.

- 	Very common CBMs like Power Storages, most generators, built-in utilities like bionic flashlight/lighter/voltmeter/toolset, Cable Chargers are "generic" and found basically anywhere. UPS and the bionic UPS are local-dimension so also go here or maybe industrial.
- 	Mining/industry/environmental protection related CBMs like Hydraulic Muscles, Skeletal Bracing, Air Filtration, Dielectric Caps, Sonic Resonator are industrial brand. 
- 	Concealed weaponry like the Shotgun Arm, Bionic Claws, Monomolecular Blade are Black Ops. As are Facial Distortion and Subdermal Carbon Filament.
- 	Low power/passive defensive/offensive CBMs are Military brand, e.g. Alloy Platings, Targeting System, Stat Aug CBMs. Can't trust grunts with complicated shit.
- 	Medical CBMs are medical. Leukocyte Breeder, Soporific Induction, Intravenous Needletip, etc.
- 	Weird Shit is either Exodii or HUB/LAB brand or both (exotic brand?). Micro Reactor, Probability Travel, Active Defense System, Uncanny Dodge, Time Dilation, etc. Power Storage II could also be Exodii/HUB? Maybe Exodii or HUB uses Power Storage II as quest rewards? Remove Power Storage II, add Exodii Power Storage and HUB Power Storage?
- 	Some stuff is Exodii-only for lore reasons. Anything that replaces limbs wholesale is Exodii; Fusion Arm Gun, Tank Legs. Sensory Dulling is Exodii-only for lore reasons. Bionic powered tool mods separate from UPS mods and the UPS bionic should exist for lore reasons, the UPS standard is local-dimension.

	
3. What do the Exodii want? What would they want, but don't yet know that they want?

- 	NECESSARY: A way to have different NPCs or factions have different barter values for different items or item categories.
- 	At first they want local CBMs, guns, and energetic materials. Liquid fuels, explosives, plutonium, ammo, that sort of thing.
- 	Once they figure out that tech here is close to Exodii peer and thus significantly more advanced than they were expecting, they want books. Do the locals know anything that we don't? You don't survive this long without being capable of humility when necessary.
- 	They want the Haber-Bosch process (fixation of ammonia from atmospheric nitrogen, portable from DDA). Either they didn't discover it or they lost it along the way. They don't want it for fertilizer, they have more than enough food from algae tank farming, they want to be able to make explosives out of air and water. The Exodii certainly have more refined, powerful, and stable explosives, but none that they can pull out of thin air and a pittance of hydrocarbons. ANFO isn't the best explosive but it is an acceptable one and mass manufacturable in a way that the vast majority of others aren't. For the Exodii, with their highly advanced nuclear reactors and mining equipment, it's practically free.
- 	The Exodii will pay a great amount for a functional example of a Haber-Bosch process machine. Perhaps even full Citizenship and a ride out of here.
- 	Along the way they can have side quests to clear out Exodii dungeons. Showcase new enemies and loot, opportunities for Exodii CBMs without trading.

4. Lore notes, stupid speech patterns, other quest ideas.

- 	The Exodii have a near-perfect translator. How did they make it? They're not gonna tell you until they're certain you can be trusted, because they made it out of a dozen feral human brains and a few of their drone workers (originally made from Exodii ferals from their home dimension). Outsiders tend to freak out about that sort of thing. This means you don't have to learn to speak alternate dimension middle-english or make up a bizarre fictional language to write quests and NPCs for the Exodii.
- 	Exodii either just use regular tools/weaponry or have purpose-built body replacement models for worker/combat drones. They don't have any need for CBMs that are basically just a tool or weapon that doesn't use encumbrance or volume. That's entirely a local thing.
- 	How do they install the Sensory Dulling CBM in a new Exodii recruit that won't already have it installed? With gritted teeth, local-dimension painkillers, and heavy restraints. It's considered a right of passage.
- 	Their currency is bionic power. Anything that can be fed into a bionic generator for power is considered currency equivalent to the amount of bionic power it would generate, however much that is actually worth to the Exodii.
- 	They don't use swappable batteries anymore, all of their stuff is bionic powered. They don't want batteries for cultural reasons (primitive and fiddly) and they'd never get more of them anyway.
- 	They don't have solar panels. Someone wants a Solar Panel CBM for infinite free money, is ultimately disappointed that the output is so low.
- 	They don't have superalloy. They're very interested in superalloy. The Exodii can't mass manufacture it so it's not suitable for their purpose-built combat drone bodies, but it'd be nice for Citizens to have and has other applications.
- 	Citizens are the Exodii who aren't drones. Drones are public property. Hints of robo-slavery-communism?
- 	Exodii eat algae sludge and will sell you algae sludge. It tastes awful but it's very efficient in both Quench and Calories. They'll also sell you the Taste Modifier CBM.

**Comment by ZChris13** [comment]
thank you for writing all this, it was fun to read, but I agree nobody's going to put the work in on this

**Comment by Manenderrr** [comment]
I'm an absolutely random person and this is a necropost, but why the tainted part

**Reply by chaosvolt** [reply]
As in, the perception among players is that the exodii exist primarily to gatekeep content that used to be obtainable as a normal part of the game world without interacting with a special faction, such as CBMs and survivor suits. Any PR that attempts to implement "exodii but better" in BN is going to innately be associated with DDA's implementation and likely will be viewed negatively as a result.

**Reply by Manenderrr** [reply]
Oh thanks for the answer
Since that's related, the Exodii guns make me think of a dieselpunk-ish faction, but I have a very rudimentary knowledge of C++ and no knowledge of the codebase whatsoever, and still don't even know how to play either cataclysm variation properly, so I'm not sure if I can even help with such things

**Reply by RosieMcCattail** [reply]
This is literally the first time I hear the word Exodii, where do I read more???

**Reply by Manenderrr** [reply]
I have no idea where to read, but they are a friendly "alien" cyborg faction in CDDA who travel across reailities to not be eaten by the zombie slime thing (I forgot its name lol) and "the Enemy"
I've encountered their outposts (called "unusual stone barn" on the map) or whatever that is near the starting evac shelter pretty often, there is a text sign in front of each of them that adds a mission to reach their base (which always spawned in 80-100 tiles away from the outpost for me), where the trader NPC is the main source of the lore about them
If that matters, his dialogues have plenty of intelligence checks that give more details about their lore

**Reply by RosieMcCattail** [reply]
Is this a DDA only thing? I don't play DDA

**Reply by Manenderrr** [reply]
Yes, it's not in BN (at least yet)

**Comment by Manenderrr** [comment]
Exodii could also have escapees from other worlds (because they know how to make unique tech and that that should be worth the effort for carrying them around), which would allow organically adding a very wide variety of strange things that don't make much sense (like, again, a different flavour of sci fi or maybe even something out of fantasy side of things) and, maybe, it would also add some kind of more complex relationship between all the factions (since IMAO Cataclysm lore really does feel like "ALL FICTIONAL REASONS FOR AN APOCALYPSE AT ONCE BOTTOM TEXT" and could be deeper)
